source/index.html

```diff
@@ -4,17 +4,186 @@
+    <!--!временно выставил размеры мобильной версии в стилях body для визуального удобства-->
+    <!--ХЕДЕР-->
     <header class="main-header">
-      <a class="main-header__logo" href="#">
-        <img class="main-header__logo-image" src="#" width="#" height="#"
-          alt="логотип магазина.">
-      </a>
+      <div class="main-header__wrapper">
+        <!--ЛОГО-->
+        <a class="main-header__logo" href="#">
+          <img
+            src="img/logo-mobile.svg"
+            width="33" height="38"
+            alt="логотип магазина.">
+        </a>
+        <a class="main-header__logo-text" href="#">
+          <img src="img/logo-text.svg"
+            width="101" height="18"
+            alt="Кэт энерджи.">
+        </a>
+        <!--НАВИГАЦИЯ-->
+        <nav class="main-nav">
+          <!--<button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>-->
+          <!--! ШАБЛОН ДЛЯ ВЕРСТКИ БУРГЕР_МЕНЮ-->
+          <input class="burger-checkbox" type="checkbox" id="burger-checkbox">
+          <label class="burger" for="burger-checkbox"></label>
+          <div class="main-nav__wrapper">
+            <!--! Q не блокирует ли враппер действие кнопки бургер на список меню?-->
+            <ul class="main-nav__list">
+              <li class="list__item list__item--active">
+                <a href="#">главная</a>
+              </li>
+              <li class="list__item">
+                <a href="catalog.html">каталог продукции</a>
+              </li>
+              <li class="list__item"><a href="form.html">
+                  подбор программы</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
     </header>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a
-        href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная
-        вёрстка и автоматизация</a>».</p>
-
+    <!--MAIN-->
+    <main class="main-container">
+      <h1 class="visually-hidden">Интернет-магазин "Cat Energy"</h1>
+      <!--HERO-->
+      <section class="hero">
+        <h2 class="hero__title">Функциональное питание для котов</h2>
+        <p class="hero__slogan">занялся собой? займись котом!</p>
+        <img src="img/index-can-mobile1.jpg" alt="#">
+        <a class="hero__button button--green" href="#">подобрать программу</a>
+        <!--! Q или правильнее прописать модификатор как на сайте БЭМ -  button_green ?-->
+      </section>
+      <section class="diet-program">
+        <div class="diet-program diet-program--slim">
+          <!--первый класс совпадает с классом секции, не хорошо-->
+          <h3 class="diet-program__title">похудение</h3>
+          <p class="diet-program__description">Ваш кот весит больше собаки и
+            <br>почти утратил способность лазить<br> по деревьям? Пора на диету!
+            Cat <br>Energy Slim поможет вашему<br> питомцу сбросить лишний
+            вес.</p>
```

Comment:
```suggestion
          <p class="diet-program__description">Ваш кот весит больше собаки и
            почти утратил способность лазить по деревьям? Пора на диету!
            Cat Energy Slim поможет вашему питомцу сбросить лишний
            вес.</p>
```
Не надо ставить br раньше времени. Только если без них никак. И лучше пользоваться неразрывными пробелами по правилам типографики, чем переносами. 

br оправданы в больших заголовках для дизайнерских целей, обычные тексты как ложатся в ширину, так и хорошо. 

У нас довольно строгий Pixel Perfect, поэтому, возможно, где-то они понадобятся. Но мы будет стараться обойтись без них.

Reply:
Памятуя как раз строгость Pixel Perfect'а я сразу застолбил места для переносов для визуального соответствия мобильной версии, чтобы на этом этапе не вспоминать о других способах переноса текста. Сейчас убираю.